db/schema.rb

```diff
@@ -26,6 +26,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.string "status"
     t.json "offer"
+    t.json "rejection"
```

Comment:
I'm just starting to wonder if we want the `rejection_reason` and `rejected_at` to be columns in the database. Would that make our lives easier? We can still return it as a hash in the API.

Reply:
Could be. 
Do we expect the reason and date to be updated independently?
Having separate columns would make what's going on in the service easier to understand rather than just passing `data` through.

Reply:
Not for this PR, but do we want to do the same for `offer`?

Reply:
Yeah, that could be `offer_conditions` with an array inside.

Reply:
carded: https://trello.com/c/PnFLwRMN/215-convert-offer-to-separate-db-fields